### PR TITLE
DOC-5971 - restructured to have separate Network Parameters and Avail…

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
…able Network Parameters topics.

Preview here: https://paul.preview.docs.r3.com/en/platform/corda/4.11/enterprise/network/network-parameters.html

### CODE REVIEW

File: content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+This topic lists the currently-available network parameters:
+
+
+* **minimumPlatformVersion**:
+The minimum platform version that the nodes must be running. Any node which is below this will
+not start.
+
+{{< note >}}
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md:22]: Note should be indented in line with bullet?

File: content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+To determine which *minimumPlatformVersion* a zone must mandate in order to permit all the features of Corda 4.11, see [Corda versioning]({{< relref "../cordapps/versioning.md" >}}).
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md:23]:
```suggestion
To determine which `minimumPlatformVersion` a zone must mandate in order to permit all the features of Corda 4.11, see [Corda versioning]({{< relref "../cordapps/versioning.md" >}}).

```

File: content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    parent: corda-enterprise-4-11-corda-networks-parameters
+    identifier: corda-enterprise-4-11-corda-networks-parameters-available
+tags:
+- network
+- map
+title: Available network parameters
+weight: 37
+---
+
+
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md:14]:
```suggestion
```

File: content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+This topic lists the currently-available network parameters:
+
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md:16]: Seems to be a lot of unnecessary line breaks in this file in general?

File: content/en/platform/corda/4.11/enterprise/network/network-parameters.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+
+If the node isn’t using a HTTP network map service then it’s expected the signed file is provided by some other means.
+For such a scenario there is the network bootstrapper tool which in addition to generating the network parameters file
+also distributes the node info files to the node directories.
+
+More parameters will be added in future releases to regulate things like allowed port numbers, whether or not IPv6
+connectivity is required for zone members, required cryptographic algorithms and roll-out schedules (e.g. for moving to post quantum cryptography), parameters related to SGX and so on.
+
+## Available network parameters
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/enterprise/network/network-parameters.md:43]: I don't think this needs a heading?

File: content/en/platform/corda/4.11/enterprise/network/network-parameters.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+The list of available network parameters can be found in the topic [Available Network Parameters]({{< relref "available-network-parameters.md" >}}). 
+
+## Network parameters update process
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.11/enterprise/network/network-parameters.md:47]: Should this be a child page, like "Available network parameters"?

File: content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md (hunk 2 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md]: Done

File: content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md]: Done

File: content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md (hunk 3 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md:14]: Done

File: content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md (hunk 4 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md:16]: Done

File: content/en/platform/corda/4.11/enterprise/network/network-parameters.md (hunk 5 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.11/enterprise/network/network-parameters.md:43]: Done

File: content/en/platform/corda/4.11/enterprise/network/network-parameters.md (hunk 6 above)
Comment (nadinequinr3) [reply content/en/platform/corda/4.11/enterprise/network/network-parameters.md:45]: The one child page looked odd to me :)

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**